packages/mui-base/src/Portal/Portal.test.tsx

```diff
@@ -148,27 +148,27 @@ describe('<Portal />', () => {
     }
 
     const { setProps } = render(<ContainerTest />);
-    expect(document.querySelector('#test3').parentElement.nodeName).to.equal('SPAN');
+    expect(document.querySelector('#test3')?.parentElement?.nodeName).to.equal('SPAN');
     setProps({
       containerElement: true,
       disablePortal: true,
     });
-    expect(document.querySelector('#test3').parentElement.nodeName).to.equal('SPAN');
+    expect(document.querySelector('#test3')?.parentElement?.nodeName).to.equal('SPAN');
     setProps({
       containerElement: true,
       disablePortal: false,
     });
-    expect(document.querySelector('#test3').parentElement.nodeName).to.equal('STRONG');
+    expect(document.querySelector('#test3')?.parentElement?.nodeName).to.equal('STRONG');
     setProps({
       containerElement: false,
       disablePortal: false,
     });
-    expect(document.querySelector('#test3').parentElement.nodeName).to.equal('BODY');
+    expect(document.querySelector('#test3')?.parentElement?.nodeName).to.equal('BODY');
   });
 
   it('should call ref after child effect', () => {
-    const callOrder = [];
-    const handleRef = (node) => {
+    const callOrder: string[] = [];
+    const handleRef = (node: any) => {
```

Comment:
That's the proper type here: 

```suggestion
    const handleRef = (node: Element | null) => {
```